### PR TITLE
fix(typeahead): fix scroll ref

### PR DESCRIPTION
**Pull request by jamesb3ll**
This change is [Reviewable](https://reviewable.io/reviews/qantasairways/runway/167)

### CODE REVIEW

File: src/components/Typeahead/index.js [hunk 1]
```diff
@@ -149,15 +149,16 @@ class Typeahead extends Component {
     highlightedIndex,
     selectedItem,
     inputValue,
-    menuHeight
+    menuHeight,
+    listRef
```
Comment (rachmulvey) [inline src/components/Typeahead/index.js]: you could access listRef directly from props here instead of sending in as a param

Comment (jamesb3ll) [reply src/components/Typeahead/index.js]: True! Thanks!

File: src/components/Typeahead/index.js [hunk 2]
```diff
@@ -148,16 +148,21 @@ class Typeahead extends Component {
...
-    menuHeight
+    inputValue
   ) => {
-    const { items, itemToString, fetchListOnInput, badgeToString } = this.props;
+    const {
+      items,
+      itemToString,
+      fetchListOnInput,
+      badgeToString,
+      listRef
```
[the remark this answers is not in the document]
Comment (jamesb3ll) [reply src/components/Typeahead/index.js]: It's used below on `ul` element.

**Pull request merged**